packages/paste-website/src/pages/components/time-picker/index.mdx

```diff
@@ -0,0 +1,608 @@
+| onChange? | `(event: React.MouseEvent<HTMLElement>)` |                                                                      | null      |
+| onFocus?  | `(event: React.MouseEvent<HTMLElement>)` |                                                                      | null      |
+| onBlur?   | `(event: React.MouseEvent<HTMLElement>)` |                                                                      | null      |
+| variant?  | 'default', 'inverse'                     |                                                                      | 'default' |
+
+#### formatReturnTime() Props
+
+| Prop       | Type   | Description                                                                                                           | Default |
+| ---------- | ------ | --------------------------------------------------------------------------------------------------------------------- | ------- |
+| timeValue  | string | Time value to be formatted. Must be in <inlineCode>HH:mm</inlineCode> format.                                         | null    |
+| timeFormat | string | Speficy the format in which to return the timeValue. Formats come from the <inlineCode>date-fns</inlineCode> library. | null    |
+
+#### Label Props
+
+All the valid HTML attributes for `label` are supported including the following props:
+
+| Prop          | Type                 | Description                                                               | Default   |
+| ------------- | -------------------- | ------------------------------------------------------------------------- | --------- |
+| children?     | `ReactNode`          |                                                                           | null      |
+| htmlFor       | string               | Sets the for of the label. Should match the id of `TimePicker`. Required. | null      |
+| disabled?     | boolean              | Shows the time picker is disabled.                                        | false     |
+| required?     | boolean              | Shows the time picker is required.                                        | false     |
+| marginBottom? | 'space0', 'space10'  |                                                                           | 'space10' |
+| variant ?     | 'default', 'inverse' |                                                                           | 'default' |
+
+#### HelpText Props
+
+All the valid HTML attributes (role, aria-\*, type, and so on) including the following props:
+
+| Prop       | Type                                           | Description        | Default   |
+| ---------- | ---------------------------------------------- | ------------------ | --------- |
+| children?  | `ReactNode`                                    |                    | null      |
+| marginTop? | 'space0', 'space20'                            |                    | 'space20' |
+| variant?   | 'default', 'error', 'error_inverse', 'inverse' | Changes the state. | 'default' |
```

Comment:
Not needed for this PR, but do you think we should eventually just have a link to the label and help text pages prop sections? We could do that for all form element doc pages once those are live.

Reply:
I guess it does make sense to link out to those pages for the props sections in the future, since someone who is looking there will likely have the motivation to click to another page. I do still think we should keep the rest of the help text/label info (guidance) duplicated on this page, though, for people who aren't willing to read a separate page.

Reply:
Agreed on the guidance. We can add a ticket in our backlog for the future links.

Reply:
I'll create ticket 👍